src/styles/LandingPage.module.less

```diff
@@ -3,13 +3,13 @@
 
 @banner-height: calc(85vh + @navbar-height*2);
 
-.LandingPageFullWidthImagesContainer  { 
-    min-height: 550px;
-    height: @banner-height;
-    width: 100vw;
-    object-fit: cover;
-    position: absolute;
-    margin: 0 -@screen-padding;
+.LandingPageFullWidthImagesContainer {
+  min-height: 550px;
```

Comment:
@toucham Thoughts on having min-height be a pixel value?

Reply:
I think min/max values as pixel is fine